chart-service/chart-generator.js

```
@@ -2,10 +1,105 @@
+import { createCanvas, registerFont } from 'canvas';
 import { Chart } from 'chart.js/auto';
 import ChartDataLabels from 'chartjs-plugin-datalabels';
 import express from 'express';
 import bodyParser from 'body-parser';
 import fs from 'fs';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)